app/controllers/toys_controller.rb

```diff
@@ -3,7 +3,18 @@ class ToysController < ApplicationController
   # skip_before_action :authenticate_user!, only: :index
 
   def index
-    @toys = Toy.all
+    if params[:query].present?
+      sql_query = " \
+        toys.name @@ :query \
+        OR toys.category @@ :query \
+        OR toys.description @@ :query \
+        OR users.first_name @@ :query \
```

Comment:
Do we search by user names? can we use user address instead? for city, etc.

Reply:
yeah. we working on it